crates/sargon-uniffi/src/wrapped_radix_engine_toolkit/low_level/compiled_transaction_intent.rs

```diff
@@ -0,0 +1,47 @@
+use crate::prelude::*;
+use sargon::CompiledTransactionIntent as InternalCompiledTransactionIntent;
+
+#[derive(Clone, PartialEq, Eq, uniffi::Record)]
+pub struct CompiledTransactionIntent {
```

Comment:
use `decl_conversion_tests_for!(CompiledTransactionIntent)` to generate the roundtrip conversion tests.